torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1939,6 +1939,26 @@ int listInplaceAdd(Stack& stack) {
   return 0;
 }
 
+template <class T>
+int listMulIntLeftInPlace(Stack& stack) {
```

Comment:
Why does this need to get implemented this way instead of desugaring 
```
x = [1, 2, 3]
x *= 3
``` 

to:
```
x = [1, 2, 3] * 3
```
?

Reply:
Doing it this way avoids copying the elements in the list for 1 of the iterations